spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/config/StandardizationConfigParser.scala

```diff
@@ -199,7 +211,9 @@ object StandardizationConfigParser {
         config.csvDelimiter.map(_ => unsupportedOptionError("--delimiter", csvFormatName)),
         config.csvEscape.map(_ => unsupportedOptionError("--escape", csvFormatName)),
         config.csvHeader.map(_ => unsupportedOptionError("--header", csvFormatName)),
-        config.csvQuote.map(_ => unsupportedOptionError("--quote", csvFormatName))
+        config.csvQuote.map(_ => unsupportedOptionError("--quote", csvFormatName)),
```

Comment:
`--csv-ignore-leading-white-space` to be consistent?

Reply:
Yes. Done. Funnily the executable one and in tests, I have `ignore`.